.github/workflows/scripts/check-autobump.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs')
+const path = require('path');
+
+module.exports = async ({github, context, core}, autobump_list) => {
+  // Parse event
+  const json = fs.readFileSync(process.env.GITHUB_EVENT_PATH, { encoding: "utf-8" })
+  const event = JSON.parse(json)
+  const pull = event.pull_request
+
+  // Fetch PR files
+  const files = await github.rest.pulls.listFiles({
+      ...context.repo,
+      pull_number: pull.number
+  })
+
+  const autobump_array = autobump_list.split(" ")
+
+  for (const file of files.data) {
+      if(autobump_array.includes(path.parse(file.filename).name)) {
+        github.rest.issues.createComment({
+          ...context.repo,
+          issue_number: pull.number,
+          body: `
+Hi! You have used \`brew bump-formula-pr\` on a formula that is on the autobump list.
+No need to manually bump it, BrewTestBot will automatically create an equivalent pull reques for you!
```

Comment:
```suggestion
Hi! You have used \`brew bump-formula-pr\` or `brew bump` on a formula that is on the autobump list.
There is no need to manually bump it. In future, let BrewTestBot automatically create an equivalent pull request for you!
```

Do we need to mention `brew bump` too? And, include "in future" so that it's clear that the PR can stay open.